gitenberg/library.py

```diff
@@ -10,23 +10,24 @@
 class GitbergLibraryManager(object):
     """ A god object for managing a collection of Gitberg style books
     """
-    def __init__(self, config=None):
+    def __init__(self):
```

Comment:
clone.Clonevat passes a config to GitbergLibraryManager(). should make this consistent. api change means semver should go to 0.3.0; the following comment should be updated.